sdk-tests/pom.xml

```diff
@@ -25,6 +25,8 @@
     <springboot.version>3.3.1</springboot.version>
     <logback-classic.version>1.4.12</logback-classic.version>
     <wiremock.version>3.9.1</wiremock.version>
+    <testcontainers-dapr.version>0.13.0-SNAPSHOT</testcontainers-dapr.version>
```

Comment:
Change the version update script in .github/scripts to also update this on release.